epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -154,6 +164,24 @@ public synchronized void init(ProcessingEnvironment processingEnv) {
     return types;
   }
 
+  @Override
+  public Set<String> getSupportedOptions() {
+    Set<String> options = new LinkedHashSet<>();
+    options.add(PROCESSOR_OPTION_IMPLICITLY_ADD_AUTO_MODELS);
+    options.add(PROCESSOR_OPTION_VALIDATE_MODEL_USAGE);
+    options.add(PROCESSOR_OPTION_REQUIRE_ABSTRACT_MODELS);
+    options.add(PROCESSOR_OPTION_REQUIRE_HASHCODE);
+    options.add(PROCESSOR_OPTION_DISABLE_KOTLIN_EXTENSION_GENERATION);
+    if (trees != null) {
```

Comment:
In the case where `trees` is null do we need to explicitly set the processor type as not incremental? Or is that the default (if so may still be worth a comment)?

Reply:
Right, the default is `DYNAMIC`. That said, it may just be better to error here because I'm not sure epoxy can safely proceed without it?